<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_ThorReg</class>
 <widget class="QDialog" name="Dial_ThorReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thor Read/Write Register</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Address">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FrontEnd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Length">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Read">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Write">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>141</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>70</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Length of Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Front End</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Read data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Data</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>361</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="checkBox_Actived">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>86</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Actived</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Hexa">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Hexadecimal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Address</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Read">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Write">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>521</width>
     <height>79</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>lineEdit_Address</zorder>
  <zorder>lineEdit_FrontEnd</zorder>
  <zorder>spinBox_Length</zorder>
  <zorder>textEdit_Read</zorder>
  <zorder>textEdit_Write</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_Read</zorder>
  <zorder>pushButton_Write</zorder>
  <zorder>pushButton_Close</zorder>
  <zorder>textEdit_Msg</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
